app/views/foreman_virt_who_configure/api/v2/configs/deploy_script.json.rabl

```diff
@@ -0,0 +1,3 @@
+object @config
+
+attributes :virt_who_config_script
```

Comment:
I'll get back to this one once we get #53 (if you prefer to merge #53 first), otherwise we can extend API there.